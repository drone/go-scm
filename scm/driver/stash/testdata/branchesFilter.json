{
  "size": 1,
  "limit": 25,
  "isLastPage": true,
  "values": [
    {
      "id": "refs/heads/master",
      "displayId": "master",
      "type": "BRANCH",
      "latestCommit": "11ce869211917dd65610e70fcee454943b35ac6e",
      "latestChangeset": "11ce869211917dd65610e70fcee454943b35ac6e",
      "isDefault": true
    },
    {
      "id": "refs/heads/master-patch",
      "displayId": "master-patch",
      "type": "BRANCH",
      "latestCommit": "11ce869211917dd65610e70fcee454943b35ac6f",
      "latestChangeset": "11ce869211917dd65610e70fcee454943b35ac6f",
      "isDefault": true
    }
  ],
  "start": 0
}