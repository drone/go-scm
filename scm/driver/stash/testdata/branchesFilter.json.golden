[
    {
        "Name": "master",
        "Path": "refs/heads/master",
        "Sha": "11ce869211917dd65610e70fcee454943b35ac6e"
    },
   {
        "Name": "master-patch",
        "Path": "refs/heads/master-patch",
        "Sha": "11ce869211917dd65610e70fcee454943b35ac6f"
    }
]